slack/utils.bal

```diff
@@ -134,6 +134,22 @@ isolated function getConversationInfo(http:Client slackClient, string channelId,
     }
 }
 
+isolated function getChannelName(http:Client slackClient, string channelId) returns string|error {
+    http:Response response = check slackClient->get(LIST_CONVERSATIONS_PATH);
+    json channelList = check response.getJsonPayload();
+    map<json> channelListJson = <map<json>> channelList;
+    _ = check checkOk(channelListJson);
+    json channels = check channelListJson.channels;
+
```

Comment:
Empty line?

Reply:
This is used to separate the code as there are many lines of code within that function.